src/test/java/org/apache/maven/plugins/dependency/CapturingLog.java

```diff
@@ -0,0 +1,179 @@
+package org.apache.maven.plugins.dependency;
+
+import org.apache.maven.plugin.logging.Log;
```

Comment:
I would expect the rat-plugin to complain about this. It should be

1. package
1. license
1. imports